firestore/FirestoreSwiftUIExample/Views/ContentView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ContentView.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+import Firebase
+
+struct ContentView: View {
+  @ObservedObject var restaurantListViewModel = RestaurantListViewModel()
+
+  var body: some View {
+    NavigationView {
+      List(restaurantListViewModel.restaurants) { restaurant in
+        NavigationLink(destination: RestaurantDetailView()) {
+          RestaurantItemView(restaurant: restaurant)
+        }
+      }
+      .navigationBarTitle("Friendly Eats", displayMode: .inline)
```

Comment:
Is there a particular reason you chose to use .inline?

Reply:
Just to match the UIKit app, but looking at it again, I'll remove so that the title has a more modern look